src/minio/core/command_builder.py

```diff
@@ -36,4 +36,52 @@ def build_alias_set_command(
             endpoint,
             access_key,
             secret_key,
-        ]
\ No newline at end of file
+        ]
+
+    # Policy Management Commands
+    def build_policy_command(
+        self,
+        action: PolicyAction,
+        policy_name: Optional[str] = None,
+        file_path: Optional[str] = None,
+    ) -> List[str]:
+        """Build policy management command.
+
+        Args:
+            action: Policy action to perform
+            policy_name: Policy name (required for all actions except LIST)
+            file_path: Policy file path (required for CREATE action)
+
+        Returns:
+            Command arguments list
+
+        Raises:
+            ValueError: If policy_name is not provided for actions that require it
+        """
+
+        if action != PolicyAction.LIST and not policy_name:
+            raise ValueError(f"Policy name is required for action: {action.value}")
+
+        if action == PolicyAction.CREATE and not file_path:
+            raise ValueError("File path is required for CREATE action")
+
+        cmd = [
+            "admin",
+            AdminCommand.POLICY.value,
+            action.value,
+            self.alias,
+        ]
+
+        # Add policy_name for all actions except LIST
+        if action != PolicyAction.LIST:
+            if policy_name:
+                cmd.append(policy_name)
+            else:
+                # This should never happen - bypass linter
+                raise ValueError(f"Policy name is required for action: {action.value}")
+
+        # Add file_path for CREATE action
+        if file_path and action == PolicyAction.CREATE:
+            cmd.append(file_path)
+
+        return cmd
```

Comment:
I'm guessing attach and detach aren't supported yet?

Reply:
yea - not currently with abstract methods but will be used in the next few PRs.